### ISSUE TITLE
Esri feature layers do not show up on some setups.

### ISSUE THREAD

**Issue by steveputman** [bhaskarvk/leaflet.esri#1]
Bhaskar, thanks again for the great package.

Have you had any experience with FeatureLayers not showing up on certain system setups? The code below works for me both on my Mac and on an AWS EC2 instance running Ubuntu 16.04, but it does not work on a Ubuntu 16.04 VM within the domain where the maps are hosted. I can get sample ESRI feature layers to show up on that machine, so it does not appear to be an ability to reach the map services. 

When I run the code on the problem machine, I get a blank map in the RStudio viewer, and then when I zoom in or out, I get only the basemap. 

```
library(leaflet)
library(leaflet.esri)

leaflet() %>%
  addEsriBasemapLayer(esriBasemapLayers$Gray) %>%
  addEsriFeatureLayer(url = "http://map2.blackstoneminerals.com/arcgis/rest/services/External/External_web_map_WM84_using_Quorum_Layers_tke_brandexv10/MapServer/6",
                      useServiceSymbology = TRUE,
                      weight = 0.5) %>%
  addEsriFeatureLayer(url = "http://map2.blackstoneminerals.com/arcgis/rest/services/External/External_Basin_Play_Factoidsv10/MapServer/29",
                      weight = 0.5) %>%
  addEsriFeatureLayer(url = "http://map2.blackstoneminerals.com/arcgis/rest/services/External/External_Map_Play_Trend_Factoidsv10/MapServer/27",
                      weight = 0.5)

```

**Comment by bhaskarvk** [comment]
Some things to try out on the problem m/c
- Does the error occur only in RStudio or also when you open the link in an external browser ?
- Could you fire up Javascript debug console and check out if you see any errors/warnings reported ?

**Comment by steveputman** [comment]
Ha, thanks--a great reminder to try different browsers! I normally access the RStudio server on the development machine through our corporate Citrix, which offers IE 11 as the browser (the maps were rendering the same way in both. In the console, I got an "access denied" on line 382 of esri-leaflet.js v1.0.3:

```
return g.open("GET", a + "?" + b(d), !0),
```

When I come in through Chrome on Windows in our intranet, it renders fine. Thanks for the tip! Closing out as it appears to be an IE issue (or at least an issue between ESRI's old script and IE).
